src/form/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Children, FC, useState } from 'react';
+import { Stack } from '../../layout';
+import { Radio } from '../Radio/Radio';
+
+export interface RadioGroupProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * The Radio Buttons to display in the group.
+   */
+  children: any;
+
+  /**
+   * Additional CSS classes to apply to the Radio group
+   */
+  className?: string;
+
+  /**
+   * The direction of the layout.
+   */
+  direction?: 'row' | 'column';
+
+  /**
+   * Event handler for when the radio is changed.
+   */
+  onChange: (value: any) => void;
+
+  /**
+   * Default value of the Radio Button which is checked
+   */
+  defaultValue: string;
+}
+
+export const RadioGroup: FC<RadioGroupProps> = ({
+  children,
+  className,
+  direction,
+  onChange,
+  defaultValue
+}) => {
+  const [selectedValue, setSelectedValue] = useState<any>(defaultValue);
+  const handleValueChange = value => {
+    setSelectedValue(value);
+    onChange(value);
+  };
+  const childrenArray: any[] = Children.toArray(children);
+
+  return (
+    <Stack direction={direction} alignItems="start" className={className}>
```

Comment:
I would not put any type of styling elements in this, this is mainly for control of the radio buttons.

Reply:
Removed the `direction` and `alignItems` property.